quickstarts/microsoft.netapp/anf-sap-hana/anf-sap-hana-storage/prereqs/prereq.azuredeploy.json

```diff
@@ -0,0 +1,269 @@
+            "location": "[parameters('location')]",
+            "properties": {
+                "encryption": {
+                    "keySource": "Microsoft.NetApp"
+                }
+            }
+        },
+        {
+            "type": "Microsoft.Network/virtualNetworks",
+            "apiVersion": "2020-11-01",
+            "name": "[parameters('virtualNetworks_vnet1_name')]",
+            "location": "[parameters('location')]",
+            "properties": {
+                "addressSpace": {
+                    "addressPrefixes": [
+                        "10.2.0.0/16"
+                    ]
+                },
+                "subnets": [
+                    {
+                        "name": "default",
+                        "properties": {
+                            "addressPrefix": "10.2.0.0/24",
+                            "privateEndpointNetworkPolicies": "Enabled",
+                            "privateLinkServiceNetworkPolicies": "Enabled"
+                        }
+                    },
+                    {
+                        "name": "ANFsubnet",
+                        "properties": {
+                            "addressPrefix": "10.2.1.0/24",
+                            "delegations": [
+                                {
+                                    "name": "Microsoft.Netapp.volumes",
+                                    "properties": {
+                                        "serviceName": "Microsoft.Netapp/volumes"
+                                    }
+                                }
+                            ],
+                            "privateEndpointNetworkPolicies": "Enabled",
+                            "privateLinkServiceNetworkPolicies": "Enabled"
+                        }
+                    }
+                ],
+                "enableDdosProtection": false
+            }
+        },
+        {
+            "type": "Microsoft.NetApp/netAppAccounts/capacityPools",
+            "apiVersion": "2021-08-01",
+            "name": "[concat(parameters('netAppAccounts_ANFaccount_name'), '/pool-ultra')]",
+            "location": "[parameters('location')]",
+            "dependsOn": [
+                "[resourceId('Microsoft.NetApp/netAppAccounts', parameters('netAppAccounts_ANFaccount_name'))]"
+            ],
+            "properties": {
+                "serviceLevel": "Ultra",
+                "size": 10995116277760,
+                "qosType": "Manual",
+                "encryptionType": "Single",
+                "coolAccess": false
+            }
+        },
+        {
+            "type": "Microsoft.Network/networkInterfaces",
+            "apiVersion": "2020-11-01",
+            "name": "[parameters('networkInterfaces_avgvm980_name')]",
+            "location": "[parameters('location')]",
+            "dependsOn": [
+                "[resourceId('Microsoft.Network/virtualNetworks/subnets', parameters('virtualNetworks_vnet1_name'), 'default')]"
```

Comment:
make this dependent on the vnet after you remove the subnet resources

Reply:
done